src/lib/config/matic.json

```
@@ -0,0 +1,45 @@
+{
+  "key": "137",
+  "chainId": 137,
+  "name": "Matic Mainnet",
+  "shortName": "Matic",
+  "network": "matic",
+  "unknown": false,
+  "nativeAsset": "MATIC",
+  "nativeAssetLong": "Matic",
+  "rpc": "https://polygon-mainnet.infura.io/v3/daaa68ec242643719749dd1caba2fc66",
+  "ws": "wss://rpc-mainnet.maticvigil.com/ws",
+  "explorer": "https://explorer-mainnet.maticvigil.com/",
```

Think we can replace this now?
```suggestion
  "explorer": "https://polygonscan.com/",
```